v0.1/elements/electrodeContact.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/elements/electrodeContact.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "SANDS definition of an electrode contact",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type", 
+    "@id",
+    "identifier",
+    "coordinate"
```

Comment:
According to the wiki `defineIn` is also a required field. Which one is correct?

Reply:
I will leave it out for now. I think we should give the option that the coordinates were directly defined in SANDS.  
We can add it as a required property later. Do you agree?